packages/design-system-react-native/src/components/BadgeWrapper/index.ts

```diff
@@ -1,7 +1,7 @@
 export {
   BadgeWrapperPosition,
-  BadgeWrapperCustomPosition,
   BadgeWrapperPositionAnchorShape,
 } from '../../types';
+export type { BadgeWrapperCustomPosition } from '../../types';
```

Comment:
We should be explicit about type imports and exports. This should have been picked up in our eslint configuration but it seems to be broken at the moment